Source/WebCore/platform/graphics/gstreamer/eme/CDMOpenCDM.cpp

```diff
@@ -283,27 +290,38 @@ void CDMInstanceOpenCDM::requestLicense(LicenseType licenseType, const AtomicStr
 void CDMInstanceOpenCDM::updateLicense(const String& sessionId, LicenseType, const SharedBuffer& response, LicenseUpdateCallback callback)
 {
     std::string responseMessage;
-    int ret = m_openCdmSession->Update(reinterpret_cast<unsigned char*>(const_cast<char*>(response.data())), response.size(), responseMessage);
-    GST_DEBUG("session id %s, calling callback %s message", sessionId.utf8().data(), ret ? "with" : "without");
-    if (ret) {
+    // FIXME: This is a realy awful API, why not just take a string& ?
+    int haveMessage = m_openCdmSession->Update(reinterpret_cast<unsigned char*>(const_cast<char*>(response.data())), response.size(), responseMessage);
```

Comment:
We are stepping on each other's foot here. I am reworking this because of the empty message answer. Go ahead with this, I'll rebase my stuff.

Reply:
That's great, I'll get this pushed ASAP.